test/model_level_tests/dump.sh

```diff
@@ -0,0 +1,28 @@
+pwd
+cd /ec/pdx/disks/aipg_lab_home_pool_01/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests/models/MLP/downloaded_model
+chmod +x /ec/pdx/disks/aipg_lab_home_pool_01/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests/models/MLP/getting_repo_ready.sh
+/ec/pdx/disks/aipg_lab_home_pool_01/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests/models/MLP/getting_repo_ready.sh
+cd /nfs/pdx/home/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests
+pwd
+cd /ec/pdx/disks/aipg_lab_home_pool_01/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests/models/MLP
+cd /ec/pdx/disks/aipg_lab_home_pool_01/ashahba/source_code/tensorflow/ngraph-bridge/test/model_level_tests/models/MLP/downloaded_model
```

Comment:
do we need this file?

Reply:
No, we don't. Nice catch

Reply:
A few more changes are coming to this branch